src/platform/nxp/crypto/se05x/CHIPCryptoPALHsm_se05x_utils.cpp

```diff
@@ -117,7 +118,11 @@ void se05x_delete_key(uint32_t keyid)
     smStatus_t smstatus   = SM_NOT_OK;
     SE05x_Result_t exists = kSE05x_Result_NA;
 
-    se05x_sessionOpen();
+    if (se05x_sessionOpen() != CHIP_NO_ERROR)
+    {
+        ChipLogError(Crypto, "se05x error: %s\n", "Error in session open");
```

Comment:
To save flash, embed the string in the error string, do not use a separate varargs. Please do this everywhere (there are many places). Also, please remove any `\n` seen in ChipLogError.

Reply:
Noted. Will change this.

Reply:
Updated with the changes